src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -76,11 +73,4 @@ public void setPiston(Value value) {
   public void togglePiston() {
     setPiston(m_climberSolenoid.get() == Value.kReverse);
   }
-
-  /**
-   * @return how far extended the climber is
-   */
-  public double getDistance() {
-    return (m_pot.getAverageVoltage() / Constants.AIO_MAX_VOLTAGE) * Constants.CLIMBER_POT_SCALE_VALUE;
-  }
```

Comment:
Instead of deleting this, can you replace it with the CANSparkMax's built-in encoder's value? You will also need to reset the encoder position at the start of the match. Jacob and/or James can help you figure out what the reduction constants should be.